main.go

```
@@ -0,0 +1,42 @@
+	// Initialize Database
+	database.Connect(AppConfig.ConnectionString)
+	database.Migrate()
+	
+	// Initialize the router
+	router := mux.NewRouter().StrictSlash(true)
+
+	// Register Routes
+	RegisterProductRoutes(router)
+
+	// Start the server
+	log.Println(fmt.Sprintf("Starting Server on port %s", AppConfig.Port))
```

golangci-lint (1.64.8)

S1038: should use log.Printf(...) instead of log.Println(fmt.Sprintf(...))

(gosimple)